src/api/arbo-typedefs.ts

```diff
@@ -52,8 +52,16 @@ export const arboTypedefs = `
     whoRegion: [String!]!
   }
 
+  type ArbovirusDataStatistics {
+    patricipantCount: Int!
+    sourceCount: Int!
+    estimateCount: Int!
+    countryCount: Int!
+  }
+
   type Query {
     arbovirusEstimates: [ArbovirusEstimate!]!
     arbovirusFilterOptions: ArbovirusFilterOptions
+    arbovirusDataStatistics: ArbovirusDataStatistics
```

Comment:
```suggestion
    arbovirusDataStatistics: ArbovirusDataStatistics!
```
Unless there's a specific case in which this would be undefined. NABD though.